R/db-odbc-mssql.R

```diff
@@ -68,7 +68,28 @@
       as.double     = sql_cast("NUMERIC"),
       as.character  = sql_cast("VARCHAR(MAX)"),
       log           = sql_prefix("LOG"),
+      nchar         = sql_prefix("LEN"),
+      atan2         = sql_prefix("ATN2"),
+      substr        = sql_prefix("SUBSTRING"),
+      ceil          = sql_prefix("CEILING"),
       ceiling       = sql_prefix("CEILING"),
+      # Casting as BIT converts the Integer result into an actual logical
```

Comment:
Yes, it does work in the WHERE statement, but it doesn't seem to work if used in the SELECT statement

```
Select * from mtcars where mpg is null
```

```
0 rows | 1-10 of 12 columns
```